pkg/plugins/gateway/cloudrun/http.go

```diff
@@ -47,24 +47,37 @@ func middleware(ctx *fasthttp.RequestCtx, pool worker.WorkerPool) bool {
 	if err := json.Unmarshal(bodyBytes, &pubsubEvent); err == nil && pubsubEvent.Subscription != "" {
 		// We have an event from pubsub here...
 
+		topicName, ok := pubsubEvent.Message.Attributes["x-nitric-topic"]
+		if !ok {
+			fmt.Println("Could not find topicName for event")
+			ctx.Error("Could not find topicName for event", 400)
+			return true
```

Comment:
Looks great, just one question. Should this return false if you don't want the event to continue processing?

Reply:
@tjholm can you chime in here? I thought you suggested this?

Reply:
correct, if we've already delegated a HTTP response (like using `ctx.Error`) then we shouldn't have the base plugin continue processing. If you want the base plugin to attempt to continue searching for a worker to handle the request you can directly return true, but we shouldn't also return a response).